### PR TITLE
[14.0][MIG] purchase_substate

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration.

Depends on:
- [x] https://github.com/OCA/server-ux/pull/366

### CODE REVIEW

**Comment by blopa98-S73** [comment]
@kittiu It would be interesting to add the field in the tree view

**Comment by RabbitJon-S73** [comment]
@kittiu any plan for this module?

**Comment by kittiu** [comment]
Sorry for the delayed, now rebased.

File: purchase_substate/data/purchase_substate_mail_template_data.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <field name="name">In delevery</field>
+        <field name="model_id" ref="purchase.model_purchase_order" />
+        <field name="subject">Reception of ${object.name}</field>
+        <field name="use_default_to" eval="True" />
+        <field name="body_html" type="html">
+            <div>
+    Dear ${object.partner_id.name or 'customer'},<br />
+    Thank you for your order.<br />
+        You order is being prepared for delivery.
+        You will recive a delivery tracking link to follow your delevery.
```
Comment (leemannd) [inline purchase_substate/data/purchase_substate_mail_template_data.xml]:
```suggestion
        You will receive a delivery tracking link to follow your delivery.
```

**Comment by kittiu** [comment]
@TheerayutEncoder please attend this for me. Thanks.

File: purchase_substate/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This module allows to add a substate to purchase order.
+For each purchase order state you can define a substate.
+We this module you can define substate which allow you to extend purchase workflow.
```
Comment (TheerayutEncoder) [inline purchase_substate/readme/DESCRIPTION.rst]:
```suggestion
With this module you can define substate which allow you to extend purchase workflow.
```

**Review by TheerayutEncoder — APPROVED** [review]
LGTM :+1:

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**